### PR TITLE
Empty and null uint256 values show up as 0x

### LINKED ISSUE #4769: Empty and null uint256 values show up as 0x
### Description

Contract state changes and contract state REST APIs show uint256 values as `0x` when the data is  `null` or an empty byte array. Instead, an empty byte array represents zero (since consensus nodes remove leading zeros to save on bytes) so should be shown as `0x0000000000000000000000000000000000000000000000000000000000000000`. For `null` db values we should show `null` in the REST API. This applies to `slot`, `value_read`, `value_written`, and `value` fields in those APIs.

### Steps to reproduce

https://testnet.mirrornode.hedera.com/api/v1/contracts/0.0.48679581/results/1666888106.746168003

### Additional context

_No response_

### Hedera network

previewnet

### Version

v0.67.1

### Operating system

_No response_

### PR DESCRIPTION
**Pull request by natanasow**
Signed-off-by: nikolay <n.atanasow94@gmail.com>

**Description**:

**Related issue(s)**:

Fixes #4769

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
Looks good, just requesting some additional test coverage:

* Modify an existing /results/{id} spec to have an unpadded zero slot and unpadded zero value_read
* Modify an existing /state spec to have an unpadded zero value (slot can't be null and will be zero padded already).

File: hedera-mirror-rest/utils.js [hunk 1]
```diff
@@ -885,6 +885,21 @@ const addHexPrefix = (hexData) => {
   return hexString.substring(0, 2) === hexPrefix ? hexString : `${hexPrefix}${hexString}`;
 };
 
+/**
+ * Pads all non-null arrays to 0x-prefixed 64 characters hex string and pass the null values as null
+ * @param val
+ * @returns {String|null}
+ */
+const toUint256 = (val) => {
+  if (val == null) {
```
Comment (edwin-greene) [inline hedera-mirror-rest/utils.js:894]: Could switch to `_.isNil` to catch null/undefined.

File: hedera-mirror-rest/utils.js [hunk 2]
```diff
@@ -885,6 +885,21 @@ const addHexPrefix = (hexData) => {
...
+    return null;
+  }
+
+  val = val.length ? val : Buffer.alloc(1);
```
Comment (xin-hedera) [inline hedera-mirror-rest/utils.js:898]: just return a constant `0x0000000000000000000000000000000000000000000000000000000000000000` if val is an empty array so can avoid allocating a 1 byte buffer and calling `toHexString`

**Review by xin-hedera — CHANGES_REQUESTED** [review]

**Review by xin-hedera — APPROVED** [review]

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Pull request merged**